deep4cast/validators.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""Validation module for evaluating forecasters.
+
+This module provides access to validation classes that can be used to
+evaluate forecasting performance.
+
+"""
+import numpy as np
+from . import metrics
+
+
+class TemporalCrossValidator():
+    """Temporal cross-validator class.
+    This class performs temporal (causal) cross-validaton similar to the
+    approch in  https://robjhyndman.com/papers/cv-wp.pdf.
+
+    :param forecaster: Forecaster.
```

Comment:
Specify the type directly in the code. Python allows and encourages that. That way, IDE can do static analysis better.